src/commands/rename_file.rs

```diff
@@ -69,7 +69,19 @@ pub fn rename_file_append(
     keymap_t: &AppKeyMapping,
 ) -> JoshutoResult {
     if let Some(file_name) = _get_current_file_name(context) {
-        let (prefix, suffix): (String, String) = match file_name.rfind('.') {
+        let (prefix, suffix) = (format!("rename {}", file_name), "".to_string());
+        command_line::read_and_execute(context, backend, keymap_t, &prefix, &suffix)?;
+    }
+    Ok(())
+}
+
+pub fn rename_file_append_base(
+    context: &mut AppContext,
+    backend: &mut AppBackend,
+    keymap_t: &AppKeyMapping,
+) -> JoshutoResult {
+    if let Some(file_name) = _get_current_file_name(context) {
+        let (prefix, suffix): (String, String) = match file_name.find('.') {
```

Comment:
Previously it is `rfind`, which will find the **last** matching pattern. I changed it to `find` to find the **first** pattern. It will be useful when handling file names with multiple suffix such as `a.tar.gz`:

```text
previous:	a.tar.gz
		    ⇑

now:		a.tar.gz
		⇑
```

Reply:
Not sure how I feel about this because sometimes files will contain periods in between that aren't necessarily related to the file extension.
It might be better to just `rfind` and `.tar.*` stuff just be categorized as a outlier.

For example, if a filename is a JWT token-like name 
```
eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c.json
```
the cursor will be very far from the real extension.
At least with `rfind` the cursor is fairly close to the real extension (even for `.tar.*`)

Reply:
You're right. I will change it back soon.